### DISCUSSION TITLE
Fetcherror

### DISCUSSION THREAD

**Discussion by sbprojects** [mattwebbio/orbital-sync#274] [Q&A]
### What happened?

With a fresh install for both pihole (docker tag 2023.03.1) and orbital sync I get the following error:

orbital-sync_1  | 5/6/2023, 8:15:48 AM: ➡️ Signing in to https://10.80.1.2:8053/admin...
orbital-sync_1  | 5/6/2023, 8:15:48 AM: ⚠ Failure: An unexpected error was thrown:
orbital-sync_1  | - FetchError: request to https://10.80.1.2:8053/admin/index.php?login failed, reason: write EPROTO 483905578B7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:

I use the same docker-compose file as shown in the example with only one secondary server:

version: '3'
[services:
  orbital-sync:
    image: mattwebbio/orbital-sync:1
    environment:
      PRIMARY_HOST_BASE_URL: 'https://10.80.1.2:8053'
      PRIMARY_HOST_PASSWORD: 'Secret'
      SECONDARY_HOST_1_BASE_URL: 'https://10.80.1.3:8053'
      SECONDARY_HOST_1_PASSWORD: 'Secret'
      INTERVAL_MINUTES: 30

Both piholes can be accessed with a web browser.
What am I doing wrong?

### Version

latest

**Comment by eko-hamburg** [comment]
I have a similar problem.
Log:
9/12/2024, 4:13:27 PM: ➡️ Signing in to http://xxx.yyy.zzz.aaa/admin...
9/12/2024, 4:13:27 PM: ⚠ Failure: An unexpected error was thrown:
- FetchError: request to https://xxx.yyy.zzz.aaa/admin/index.php?login failed, reason: unable to verify the first certificate
It seems to be related to my certificate that I use for pihole. I use a own internal CA. How can I add this CA-Certificate to the orbital-sync docker?
Thank you for your help.
